tests/unit/sagemaker/image_uris/test_huggingface_llm.py

```diff
@@ -21,9 +21,11 @@
 TEI_VERSIONS_MAPPING = {
     "gpu": {
         "1.2.3": "2.0.1-tei1.2.3-gpu-py310-cu122-ubuntu22.04",
+        "1.4.0": "2.0.1-tei1.4.0-gpu-py310-cu122-ubuntu22.04",
```

Comment:
Not related to this PR, I saw one critical vuln in the just-released TEI image: https://tiny.amazon.com/spcoq4lc/IsenLink

Reply:
Might be a false positive. I double checked our scanner in gamma is passing. or probabaly is CVE is ignored by our gamma scanner. I will double check